core/drivers/imx_lpuart.c

```diff
@@ -63,10 +64,72 @@ KEEP_PAGER(imx_lpuart_ops);
 void imx_uart_init(struct imx_uart_data *pd, paddr_t base)
 {
 	pd->base.pa = base;
+	pd->base.va = 0;
```

Comment:
Not needed, it should be safe to assume that *pd is zero initialized (it is either a static struct, or allocated below where we can use `calloc()`).

Reply:
Same comment applies to `imx_uart.c`, too.

Reply:
I will fix it.